FSharp.Profiles.props

```diff
@@ -22,6 +22,10 @@
     <OtherFlags>$(OtherFlags) /langversion:preview</OtherFlags>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(BUILDING_USING_DOTNET)' == 'true'"> 
+      <CheckNulls>false</CheckNulls>
+  </PropertyGroup>
+
```

Comment:
Turning it off currently for the plain builds, so we don't get mismatches with defines we have.